src/split-panel/styles.scss

```diff
@@ -215,6 +218,7 @@ $app-layout-drawer-width: calc(#{awsui.$space-layout-toggle-diameter} + 2 * #{aw
   :not(.drawer-closed) > .drawer-content-bottom > & {
     background-color: awsui.$color-background-layout-panel-content;
     border-bottom: awsui.$border-panel-header-width solid awsui.$color-border-divider-default;
+    margin-top: -$slider-width - 2px; // the height of the slider
```

Comment:
> $slider-width
> // the height of the slider

Something does not match up here

Reply:
$slider-width is used to define the height of the slide when it is bottom split panel. I will update the comment to make it more clear. 
https://github.com/cloudscape-design/components/blob/6fd6fd1a8f92fc6a941ef9aba79bdc380eda12f4/src/split-panel/styles.scss#L159

Reply:
I think, there is something wrong with `$slider-width` if it is used in both vertical and horizontal directions, it should be called `$slider-size`, but ok, it is for another time.